### PR TITLE
Enhance rsyslog documentation

### PR DESCRIPTION
**Pull request by ka0ula**
- Remove the huge titles
- Improve english

### CODE REVIEW

File: docs/xdr/features/collect/ingestion_methods/rsyslog.md [hunk 1]
```diff
@@ -98,134 +98,134 @@ To receive and process Windows logs, follow these steps:
...
+	The `$hostname` in the `if condition` refers to the `%hostname%` value in the syslog header. Indeed, depending of your network, the syslog `%hostname%` can be an FQDN, an IP address (with or without NAT) or the real Hostname of the source machine.
 
-7. Comment the lines of the file "/etc/rsyslog.d/00-testing.conf"
+	7. Comment the lines of the file "/etc/rsyslog.d/00-testing.conf"
 
-```bash
-# template(name="SEKOIAIOTesting" type="string" string="<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% %procid% LOG [SEKOIA@53288 intake_key=\"DO_NOT_CHANGE\"] %msg%\n")
+	```bash
+	# template(name="SEKOIAIOTesting" type="string" string="<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% %procid% LOG [SEKOIA@53288 	intake_key=\"DO_NOT_CHANGE\"] %msg%\n")
 # *.* /var/log/testing.log;SEKOIAIOTesting
```
Comment (squioc) [inline docs/xdr/features/collect/ingestion_methods/rsyslog.md:182]: Due to the indentation, this block is not well rendered

**Review by squioc — CHANGES_REQUESTED** [review]

File: docs/xdr/features/collect/ingestion_methods/rsyslog.md [hunk 2]
```diff
@@ -2,295 +2,324 @@
...
-```bash
-sudo vim /etc/rsyslog.d/00-testing.conf
-```
+	```bash
+	sudo tcpdump -i <change_with_interface_name> -c10 -nn src <IP_OF_THE_SOURCE> -vv
 
-- Ensure this file will contain the following information only
+	```
+	!!!tip
+		Use `ip addr` command to find the relevant information to relace `<change_with_interface_name>`.
```
Comment (squioc) [inline docs/xdr/features/collect/ingestion_methods/rsyslog.md:106]:
```suggestion
		Use `ip addr` command to find the relevant information to replace `<change_with_interface_name>`.

```

**Review by squioc — CHANGES_REQUESTED** [review]

**Review by squioc — APPROVED** [review]
Nice work :+1:!!

**Comment by ka0ula** [comment]
@squioc thanks for the review! I hope it helps 😊

**Pull request merged**